ui/admin/app/components/form/storage-bucket/minio/index.hbs

```diff
@@ -166,6 +166,7 @@
   {{! bucket_name }}
   <Hds::Form::TextInput::Field
     name='bucket_name'
+    @id='bucket_name'
```

Comment:
The usage of ID is to could use standard selectors also for error messages.

Once we assign attribute `@id`, such id will be used within the `<input id='bucket_name'>` but also it will be added to the error message as follows: `error-bucket_name`.  This pattern allows using a standard and descriptive selector to select a specific field and its error message.

Reply:
nice find! That should make tests against error states for our inputs much easier.

Reply:
Should we move away from using IDs after the discussion in https://github.com/hashicorp/boundary-ui/pull/2278?

Reply:
@ZedLi 
Ups!! totally forgot in this PR I was using those too!!
Sure!! as per discussion on #2278 I will proceed to rely on `data-test-*` for this PR, I will require a review from you and @cameronperera as soon I got that in 😉 
Thanks for the feedback and sorry for the back/forth.

Reply:
No problem!! Just wanted to make sure we were on the same page

Reply:
@ZedLi 
I applied changes 😉